build.gradle

```diff
@@ -11,8 +11,11 @@ sourceCompatibility = JavaVersion.VERSION_1_6
 dependencies {
   compile 'io.dropwizard.metrics:metrics-core:3.1.0'
   compile 'com.newrelic.agent.java:newrelic-api:3.10.0'
-
+  compile 'com.google.guava:guava:18.0'
```

Comment:
Oh i see. Yes, that could be pretty good. I'm not familiar with gradle so it will take me a while to do that, in case you are able to do it, go ahead :dancer: